### ISSUE TITLE
setbanked loop when booting on Thinkpad T440p

### ISSUE THREAD

**Issue by fhs** [Harvey-OS/harvey#977]
It gets stuck printing this:
```
setbanked: ra 0x1800 rb 0x0 sz 2 value 0x1
```

Video of the boot here: https://youtu.be/CgPTOjpBCgc

**Comment by rminnich** [comment]
the set banked is not the problem. The problem is it does not have a network. 

Is there anyway you can boot with the rc option, not the network option, so we can poke around a bit. But the network is your issue.

**Comment by fhs** [comment]
I can boot with the rc option, but I don't know how to poke around with all the basic commands (cat, ls, etc.) missing.

I have an i217:
```
0.25.0: net  02.00.00 8086/153a   6 0:f0600000 131072 1:f063f000 4096 2:00003081 32
        Intel Corporation Ethernet Connection I217-LM
```
which is supported by the same driver as i219, so I think I'll give PR #980 a try later today.

**Comment by gmacd** [comment]
/boot/ls /boot/cat etc😁

**Comment by orangecms** [comment]
I have a patch for the verbose spewing; see #982

**Comment by orangecms** [comment]
Could you also drop some `lspci` logs on #984 please? :)

**Comment by orangecms** [comment]
The numerous `setbanked` logs are now fixed with #982 and the network issue is tracked in #984, so we can close this, right?

**Comment by fhs** [comment]
> The numerous `setbanked` logs are now fixed with #982 and the network issue is tracked in #984, so we can close this, right?

Sounds good.
